src/Definition/Common.php

```diff
@@ -24,11 +32,17 @@ class Common
      * @param string $length
      * @param bool $nullable
      */
-    public function __construct($type, $length = null, $nullable = true)
+
+    public function __construct($type, $length = null, $nullable = true, $default = null, $format = null)
```

Comment:
If `Redshift` and `Snowflake` classes do not support `default` and `format`, it should be avoided from the `Common` class entirely. If you want to create a dummy class for metadata operations, then go ahead! Maybe the library is poorly structured. `Common` is something all classes should extend, but should not be used alone anywhere. Maybe it shouldbe `abstract`.